finalPrep/modZeroHero.js

```diff
@@ -56,29 +129,89 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "Ivan the Terrible",
+  smell: "ghastly",
+  weight: 500,
+  citiesDestroyed: ['Paris', 'Miami', 'Dallas'],
+  luckyNumbers: [10, 33, 55, 75],
+};
+
+var monsterAddress = {
```

Comment:
You have declared this as a new variable, but the address can be within your scaryMoster object, the value of that `address` key will just be another object that looks like the one you have in a separate variable.